test/extensions/filters/http/fault/fault_filter_test.cc

```diff
@@ -149,14 +149,14 @@ class FaultFilterTest : public testing::Test {
   void TestPerFilterConfigFault(const Router::RouteSpecificFilterConfig* route_fault,
                                 const Router::RouteSpecificFilterConfig* vhost_fault);
 
+  Stats::IsolatedStoreImpl stats_;
```

Comment:
out of curiosity, any purpose behind reordering this member?

Reply:
Yes: the stat-store need to outlive the config since we use the symbol table in the store to remember some stat-names in the config.